mappymatch/utils/plot.py

```diff
@@ -178,3 +192,91 @@ def plot_map(tmap: NxMap, m=None):
         ).add_to(m)
 
     return m
+
+
+def plot_match_distances(coord_df):
+    # todo MatchResult is not a defined Element, removed from the args list above (was (matches: MatchResult)).
+    # build and display plots here.
+    """
+    Summary: Plot the points deviance from known roads with matplotlib.
+
+    Issues:
+        we have two different dataframes one labeled with gdf and the other with df. --> to resolve this, the gdf labeled data frames have been changed to df labels.
+
+    Args:
+        coord_df (pandas dataframe): coords of guessed points in the area.
+    """
+
+    # Road Data Frame Section
```

Comment:
It's generally ok to just delete commented-out code, because it can always be retrieved from the git history, and it improves the readability of the live code.

Reply:
I agree here, we should probably just remove all the commented code before merging.